docs/en/ingest-management/fleet/air-gapped.asciidoc

```diff
@@ -0,0 +1,126 @@
+or deploy your own instance of the {package-registry}.
+
+[discrete]
+[[air-gapped-proxy-server]]
+== Use a proxy server
+
+If you can route traffic to the public endpoint of the {package-registry} through a network gateway, there is a property in {kib} that
+can orchestrate to use a proxy server:
+
+[source,yaml]
+----
+xpack.fleet.registryProxyUrl: your-nat-gateway.corp.net
+----
+
+[discrete]
+[[air-gapped-diy-epr]]
+== Host your own {package-registry}
+
+If routing traffic through a proxy server is not an option, you can host your own {package-registry}.
+
+The {package-registry} can be deployed and hosted on-site using one of the
+available Docker images. These docker images include the {package-registry} and
+a selection of packages.
+
+There are different distributions available:
+
+* {version} (recommended): +docker.elastic.co/package-registry/distribution:{version}+ - Selection of packages from the production repository released with the {version} version of the {stack}.
+* production: `docker.elastic.co/package-registry/distribution:production` - Packages available in the production registry (https://epr.elastic.co).
+* staging: `docker.elastic.co/package-registry/distribution:staging` - Packages available in the staging registry (https://epr-staging.elastic.co). These packages may be pending of validation.
+* snapshot: `docker.elastic.co/package-registry/distribution:snapshot` - Packages under development.
+
+ifeval::["{release-state}"=="unreleased"]
+[WARNING]
+====
+Version {version} of the {package-registry} distribution has not yet been released.
+====
+endif::[]
+
+To update the distribution image, re-pull the image and then restart the docker container.
+
+Every distribution contains packages that can be used by different versions of
+the {stack}. The {package-registry} API exposes a {kib} version constraint that
+allows for filtering packages that are compatible with a particular version.
+
+NOTE: These steps use the standard Docker CLI, but you can create a Kubernetes manifest
+based on this information.
+These images can also be used with other container runtimes compatible with Docker images.
+
+1. Pull the Docker image from the public Docker registry:
++
+["source", "sh", subs="attributes"]
+----
+docker pull docker.elastic.co/package-registry/distribution:{version}
+----
++
+2. Save the Docker image locally:
++
+["source", "sh", subs="attributes"]
+----
+docker save -o package-registry-{version}.tar docker.elastic.co/package-registry/distribution:{version}
+----
++
+TIP: Check the image size to ensure that you have enough disk space. 
+
+3. Transfer the image to the air-gapped environment and load it:
++
+["source", "sh", subs="attributes"]
+----
+docker load -i package-registry-{version}.tar
+----
+
+4. Run the {package-registry}:
++
+["source", "sh", subs="attributes"]
+----
+docker run -it docker.elastic.co/package-registry/distribution:{version}
```

Comment:
If my testing is correct, this will not expose port of the registry outside of the container, so Kibana won't be able to access the registry. Please compare with next example with TLS configuration with `-p` option. Can you please check?

Reply:
`-p 8080:8080` added, thanks for the feedback!